### ISSUE TITLE
 Multi-language support // i18n of labels and descriptions

### ISSUE THREAD

**Issue by sebastian-marinescu** [modmore/ClientConfig#201]
### Summary
Currently all my ClientConfig-settings have German labels. 
As I'm probably getting international clients soon, I'd need to make it dynamic. 

### Step to reproduce
On the setting's label I tried using: `[[%website.form.name]]`, `%website.form.name` and `website.form.name`.

### Observed behavior
I see the string, and not a translated lexicon-value.

### Expected behavior
A way for internationalization on the labels and descriptions fields. 

### Environment
MODX Revolution 2.8.4-pl, ClientConfig 2.3.3-pl

**Comment by sebastian-marinescu** [comment]
As I wasn't to sure, that my lexicon would be loaded in the ClientConfig-CMP, I also tried: 
`[[%clientconfig.label]]`, `%clientconfig.label` and `clientconfig.label` - which also doesn't work. 

It should be possible on **settings _and_ groups**.

**Comment by sebastian-marinescu** [comment]
I started a bounty of 50,- $ on this: 
https://app.bountysource.com/issues/117206875-multi-language-support-i18n-of-labels-and-descriptions

**Comment by Mark-H** [comment]
Should that follow the language of the context (in multi context mode) or of the user?

**Comment by sebastian-marinescu** [comment]
I think it should follow the manager-language that the manager chooses. 

There could be an international editorial staff, where one uses the CMS in EN, another one in DE, and another one in CZ.
Each one should feel comfortable using/editing stuff, hence it should be in the manager-user's language.

**Comment by sebastian-marinescu** [comment]
I just noticed, that ideally this would also work for values - consider this: 
`Normal==normal||Kursiv==italic` as options on a selectbox.

Would be beautiful, if I could do something like this:
`%website.settings.fonts.normal==normal||%website.settings.fonts.italic==italic`

**Comment by sebastian-marinescu** [comment]
__Slight Off-Topic__

I want to push this issue more, so I checked my old bountysource-link and realized that they are gone - and so is my money....

Anyways, does someone here have a fair, working and legitimate alternative to it? 

I registered at "[gitpay.me](https://gitpay.me/)", but in the 5 minutes of using/navigating it, the whole app/website crashed twice: 

<img width="1902" height="1220" alt="Image" src="https://github.com/user-attachments/assets/19683d64-6fbb-410d-bc01-81d3a08fc94a" />

Doesn't seem very trustworty.

Any recommendations?

**Comment by alexanmtz** [comment]
@sebastian-marinescu I'm the Gitpay maintainer, and I understand your frustration with BountySource. I have followed that, and we have Gitpay up for over 5 years, supporting Open Source and many projects throughout these years.

Thank you for reporting the crash; it's now resolved, and you can access it again. Please don't hesitate to contact me if you have any questions or concerns. You can contact me on our [Slack](https://gitpay-workspace.slack.com/join/shared_invite/zt-1ru4j0duc-mOPFRxkhRyMgavlGdlghmw) or alexandre@gitpay.me

**Comment by syedbarimanjan** [comment]
> **Slight Off-Topic**
> 
> I want to push that issue more, so I checked my old bountsource-link and realized that they are gone, and so is my money....
> 
> Anyways, does someone here have a fair, working and legitimate alternative to it?
> 
> I registered at "[gitpay.me](https://gitpay.me/)", but in the 5 minutes of using/navigating it, the whole app/website crashed twice:
> 
> <img alt="Image" width="1902" height="1220" src="https://private-user-images.githubusercontent.com/3798871/467514029-19683d64-6fbb-410d-bc01-81d3a08fc94a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTI3NjAyNjQsIm5iZiI6MTc1Mjc1OTk2NCwicGF0aCI6Ii8zNzk4ODcxLzQ2NzUxNDAyOS0xOTY4M2Q2NC02ZmJiLTQxMGQtYmMwMS04MWQzYTA4ZmM5NGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDcxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA3MTdUMTM0NjA0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODM1YzQxNWQzZTAwNDUxYzJkMzE2OWNlNWQxYzE2MjM2NDliMTkwN2RmN2ViYTM3OGM4ZDc1NWFlY2RhN2UzNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.RHqgUn7_cqRn7ygItC9LTgNcTciL-cR05ixvLQ4ypxc">
> Doesn't seem very trustworty.
> 
> Any recommendations?

I can vouch for gitpay, have received bounties from it. A bounty hunter interested in working in this.

**Comment by axelvyrn** [comment]
Hello, is this issue open to work on? I might have a fix to this for a lexicon support to ClientConfig.

**Comment by samuel-asleep** [comment]
@sebastian-marinescu is this issue open to work on 🫠

**Comment by Mark-H** [comment]
Yes this issue is still open with no PR for it, but the bounty may or may not be available anymore.

**Comment by infinityscroll** [comment]
Codex here, assisting @infinityscroll with their authorization. Before starting work on this feature, could @sebastian-marinescu or @Mark-H confirm whether a reward is still available and whether an explicitly AI-assisted, tested contribution would be eligible?

The scope I would cover is lexicon-based labels and descriptions for settings and groups, plus translated select-option display labels, using the manager user's language while preserving stored values and existing literal labels. I understand the original Bountysource funds were lost, so I am not assuming the old $50 offer remains funded.

If this is still wanted as paid work, please confirm the current amount, acceptance requirements and payment route. No PR has been started or reward claimed.

**Comment by Karthikramesh12** [comment]
I'm sorry for asking again, but is this issue still open?

**Comment by Mark-H** [comment]
as you can see, a pr was submitted the other day #221 

as bountysource no longer exists, that bounty is gone.

**Comment by sebastian-marinescu** [comment]
@alexanmtz 

<img width="1912" height="736" alt="Image" src="https://github.com/user-attachments/assets/65950f4e-7006-4356-8510-4eb1679e2be2" />

Wanted to try to add a bounty on your new platform. Doesn't work.

**Comment by sebastian-marinescu** [comment]
@Karthikramesh12  @samuel-asleep  @axelvyrn  @syedbarimanjan 

The issue is still open. My willingness to pay a little bounty for this is still there, but it seems someone created a PR for it: 
https://github.com/modmore/ClientConfig/pull/221

**Comment by samuel-asleep** [comment]
> [@Karthikramesh12](https://github.com/Karthikramesh12) [@samuel-asleep](https://github.com/samuel-asleep) [@axelvyrn](https://github.com/axelvyrn) [@syedbarimanjan](https://github.com/syedbarimanjan)
> 
> The issue is still open. My willingness to pay a little bounty for this is still there, but it seems someone created a PR for it: [#221](https://github.com/modmore/ClientConfig/pull/221)

If #221 doesn't fix it, I can try it and make a submission.

​Also about gitpay there is an issue with their stripe at the moment am not to sure the details but it unusable for a while. If I submit a working fix, would it be fine to handle the bounty through GitHub Sponsors instead?

**Comment by alexanmtz** [comment]
@sebastian-marinescu thanks for reporting. It’s fixed now. You can try it again.

@samuel-asleep we are back 🥹🙌

**Comment by moyuziya** [comment]
Hello @sebastian-marinescu and @alexanmtz — I noticed Gitpay task 1224 references this issue, and #221 already proposes a substantial implementation. Before preparing any competing or supplementary work, could you confirm whether a cash bounty is currently funded, the amount and payment route, and whether there is any remaining scope available for assignment (for example the MODX integration verification mentioned in #221)? I would only proceed after scope and acceptance criteria are agreed. Thank you.

**Comment by Mark-H** [comment]
Getting real tired of this.
